### PR TITLE
Less Nosegrab (Balance)

### PR DESCRIPTION
**Pull request by Optimism333**
## About The Pull Request
Nukes nosegrab off the couple remaining drifter classes that had it. It is now unique to the King.

Basically finishes the work I started in #533 by removing it from mercenary classes.

### Changes ###
- Removed nosegrab from Freelancer
- Removed nosegrab from Grenzelhoft noble
- Removed nosegrab from Zybantu noble
- Removed nosegrab from Heartfelt Lord

## Why It's Good For The Game
Nosegrab is a very powerful grappling trait, and I don't believe it should be on roles dedicated to combat (Lancer) or on random noble roles from out of town. Keeping it unique to King gives players a little bonus for playing the most important in-town role in the game, and keeps it rare.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/serfs/innkeep.dm [hunk 1]
```diff
@@ -32,6 +32,8 @@
 		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
+		ADD_TRAIT(H, TRAIT_NOSEGRAB, TRAIT_GENERIC)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/serfs/innkeep.dm:35]: Why.

Comment (NPC1314) [reply code/modules/jobs/job_types/roguetown/serfs/innkeep.dm:35]: Some towner roles having a little more bite wont hurt anyone, without adding weapons/armor or combat skills this is a good way to do it.

If mercs having it didnt destroy pvp innkeep having it certainly wont, and there needs to be some carrots for towner roles, gimmicks, gameloop stuff, small powerups, 

Some sugar is needed and with serious lack of policing some of it has to be coercive power, drifters has a very easy time to bully and more or less grief townies as is. Unless policy of expecting self-policing changes this sort of thing will probably bring more good than bad.

Comment (Optimism333) [reply code/modules/jobs/job_types/roguetown/serfs/innkeep.dm]: I don't really feel like arguing about it. If someone wants to add it to Innkeeper, I am indifferent, as I have faith that our very few innkeeper players would not abuse it. 

But maybe I'm too optimistic.

Comment (Optimism333) [reply code/modules/jobs/job_types/roguetown/serfs/innkeep.dm]: Well, it's gone from the pr. That's the end of it.

**Review by NPC1314 — APPROVED** [review]
Mistake to remove the inkeep nosegrab IMO but oh well, your PR.

**Pull request merged**